hooks/tk-multi-launchapp/before_app_launch.py

```diff
@@ -27,6 +28,10 @@
 
 from ss_config.hooks.tk_multi_launchapp.before_app_launch import SsBeforeAppLaunch
 
+project = sgtk.platform.current_engine().context.project
+tk = sgtk.sgtk_from_entity(entity_type=project["type"], entity_id=project["id"])
```

Comment:
you sure tk = sgtk.platform.current_engine().context.sgtk, won't just work? not so sure we need to be instancing a new tk, we're literally in one